Failing test: Serverless Observability Functional Tests.x-pack/solutions/observability/test/serverless/functional/test_suites/rules/es_query_consumer·ts - serverless observability UI ES Query rule - consumers both logs and infrastructure privileges should open the rule creation flyout

A test failed on a tracked branch

```
JestAssertionError: expect(received).toBe(expected) // Object.is equality

Expected: true
Received: false
    at Context.<anonymous> (es_query_consumer.ts:37:41)
    at Object.apply (wrap_function.js:74:16) {
  matcherResult: {
    actual: false,
    expected: true,
    message: '\x1B[2mexpect(\x1B[22m\x1B[31mreceived\x1B[39m\x1B[2m).\x1B[22mtoBe\x1B[2m(\x1B[22m\x1B[32mexpected\x1B[39m\x1B[2m) // Object.is equality\x1B[22m\n' +
      '\n' +
      'Expected: \x1B[32mtrue\x1B[39m\n' +
      'Received: \x1B[31mfalse\x1B[39m',
    name: 'toBe',
    pass: false
  }
}
```

First failure: [kibana-elasticsearch-serverless-verify-and-promote - main](https://buildkite.com/elastic/kibana-elasticsearch-serverless-verify-and-promote/builds/4171#0198f768-529b-4463-89f9-ff344d26fa47)

<!-- kibanaCiData = {"failed-test":{"test.class":"Serverless Observability Functional Tests.x-pack/solutions/observability/test/serverless/functional/test_suites/rules/es_query_consumer·ts","test.name":"serverless observability UI ES Query rule - consumers both logs and infrastructure privileges should open the rule creation flyout","test.failCount":1}} -->

--- a/x-pack/solutions/observability/test/serverless/functional/test_suites/rules/es_query_consumer.ts
+++ b/x-pack/solutions/observability/test/serverless/functional/test_suites/rules/es_query_consumer.ts
@@ -32,6 +32,7 @@ export default ({ getPageObject, getService }: FtrProviderContext) => {
     });
 
     it('should open the rule creation flyout', async () => {
+      await testSubjects.existOrFail('createRuleButton');
       await testSubjects.click('createRuleButton');
       const isCreateRuleFlyoutVisible = await testSubjects.exists('ruleTypeModal');
       expect(isCreateRuleFlyoutVisible).toBe(true);

